core/commands/resolve.go

```diff
@@ -14,37 +13,46 @@ type ResolvedPath struct {
 	Path path.Path
 }
 
-var resolveCmd = &cmds.Command{
+var ResolveCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Gets the value currently published at an IPNS name",
+		Tagline: "Resolve all supported protocols to IPFS",
 		ShortDescription: `
-IPNS is a PKI namespace, where names are the hashes of public keys, and
-the private key enables publishing new (signed) values. In resolve, the
-default value of <name> is your own identity public key.
+There are a number of mutable protocols that can link among themselves
```

Comment:
On Thu, May 14, 2015 at 09:26:10PM -0700, Juan Batiz-Benet wrote:
> >  		ShortDescription: `
> > -IPNS is a PKI namespace, where names are the hashes of public keys, and
> > -the private key enables publishing new (signed) values. In resolve, the
> > -default value of <name> is your own identity public key.
> > +There are a number of mutable protocols that can link among themselves
> 
> `... number of mutable name protocols ...`

Added to my current reroll.